### ISSUE TITLE
[Question] - Should some of the Helm charts in this repo be moved to redhat-cop/helm-charts 

### ISSUE THREAD

**Issue by springdo** [redhat-cop/openshift-management#75]
Discuss.

**Comment by garethahealy** [comment]
Personally, I see the charts in here as cleanup/helper tasks and not charts to deploy an application which is what I see the other repo for.

**Comment by springdo** [comment]
Yeah I'd agree that they're not _application_ charts, but still useful. I added a note of their existence on the helm-repo. We did just add some of them to the 🦄🔥 project too as part of a `day2ops` set of values.
